### ISSUE TITLE
Add "manager" level permissions (add/edit feature flags, etc.)

### ISSUE THREAD

**Issue by jackiegreenfield** [PostHog/posthog#9523]
## Is your feature request related to a problem?

We want to use feature flags in place of our current internally-built feature toggles, and that's brought up a concern of accidental changes. Today, we use an in-house solution that only certain people can access. 

We have non-technical teams like Support and Compliance who will want to use Posthog for the Recordings; they won't be fully trained on how to use the rest of the product, and could accidentally mess something up.

## Describe the solution you'd like
We'd like a level of access that's somewhere between member and admin. Think of it as the kinds of things that a product manager would need to do, like adjusting which users / companies have access to feature flags.

So, basically, the idea is that the following would be restricted to this 'manager' level (and admins/owners), but _not_ members. 
- Ability to add/edit feature flags
- Ability to add/edit experiments 
- Ability to edit Project Settings
- Ability to add/remove plugins 
- Ability to add/remove global annotations (maybe?) - or maybe they can add them, but can't edit existing ones
- Ability to delete cohorts

## Describe alternatives you've considered

Training everyone to stay away from areas that they could mess up.
